client/client-multi/client-objective-c/Kaa/KaaClientFactory.h

```diff
@@ -0,0 +1,44 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
```

Comment:
I believe that copyright header formats aren't that serious issue to prevent PRs to being merged.
At this point of time license plugin is configured so each file extension has it's own copyright format thus violating it's rules is a larger issue than considering another format for any reason. If you want to introduce another format, I'd better create issue for that but for real it shouldn't any high priority until it raises any other issues. IMHO.